src/main/java/org/loose/fis/mov/controllers/LoginController.java

```diff
@@ -29,8 +29,10 @@ public void handleLoginAction(ActionEvent event) {
                 User user = UserService.login(usernameField.getText(), passwordField.getText());
                 if (Objects.equals(user.getRole(), "Admin")) {
                     changeScene(event, "mainMenuAdmin.fxml");
-                } else {
-                    loginMessage.setText(String.format("Welcome, %s %s!", user.getRole(), user.getUsername()));
+                }
+                if( (Objects.equals(user.getRole(), "Client")))
```

Comment:
Pune asta direct in else ca sa nu ne scada la testing coverage-ul pentru ca nu testam ambele branch-uri.